modules/sdk-core/src/index.ts

```diff
@@ -9,7 +9,7 @@ import { EddsaUtils } from './bitgo/utils/tss/eddsa/eddsa';
 export { EddsaUtils };
 import { EcdsaUtils } from './bitgo/utils/tss/ecdsa/ecdsa';
 export { EcdsaUtils };
-export { GShare, SignShare, YShare } from './account-lib/mpc/tss/eddsa/types';
+export { GShare, SignShare, YShare, Signature as edDSASignature } from './account-lib/mpc/tss/eddsa/types';
```

Comment:
Types should be Pascal cased (first letter should be capitalized).

```suggestion
export { GShare, SignShare, YShare, Signature as EddsaSignature } from './account-lib/mpc/tss/eddsa/types';
```

or 
```suggestion
export { GShare, SignShare, YShare, Signature as EDDSASignature } from './account-lib/mpc/tss/eddsa/types';
```